server/manager/src/main/java/org/apache/accumulo/manager/recovery/RecoveryManager.java

```diff
@@ -197,9 +197,11 @@ public boolean recoverLogs(KeyExtent extent, Collection<Collection<String>> walo
         synchronized (this) {
           if (!closeTasksQueued.contains(sortId) && !sortsQueued.contains(sortId)) {
             AccumuloConfiguration aconf = manager.getConfiguration();
+            @SuppressWarnings("deprecation")
             LogCloser closer = Property.createInstanceFromPropertyName(aconf,
-                Property.MANAGER_WALOG_CLOSER_IMPLEMETATION, LogCloser.class,
-                new HadoopLogCloser());
+                aconf.resolve(Property.MANAGER_WAL_CLOSER_IMPLEMENTATION,
+                    Property.MANAGER_WALOG_CLOSER_IMPLEMETATION),
```

Comment:
I think @milleruntime had suggested elsewhere in his review that it might not be necessary to resolve between these two, because the one being deprecated is entirely new. However, I think it probably still is necessary, because the upgrade code will read in `master.walog.closer.implementation` as `manager.walog.closer.implementation`, so it could still be in use inside the AccumuloConfiguration object. So, I think this is fine. If we want the upgrade code to do some additional translations directly to this new version, to eliminate the intermediate `manager.walog.` version, we can do that, but I don't think we should, because having the intermediate version helps users easily transition their old config files with simple `sed` commands, and that will break if we don't have a direct mapping of all `master.` props to `manager.` ones.

Reply:
> having the intermediate version helps users easily transition their old config files with simple sed commands, and that will break if we don't have a direct mapping of all master. props to manager. ones.

That is a good point. It is unfortunate that we will change a user's property to a deprecated one but having the intermediate property makes the changes more flexible. I think this is OK as long as the version that eventually drops the `walog` properties, does the replace in the upgrade code. We have become better at resolving things in the upgrade code but this is another example of why you should never skip Major or Minor versions when upgrading.